jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
+target = '/root/test-catalogs'
+
+teardown do
+  %x(rm -rf #{tmp_repo})
```

Comment:
Sure, I can go through the Ruby file utils to do this, but...

How would that make it any safer? I still need to tell it to force remove the directory.  How does testing for a directory help?  I could see this being valuable if I was deleting a file and wanted to make sure to not delete a directory.